src/react-16.3/AccountData.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/*
+ * Create component.
+ */
+
+class AccountData extends Component {
+  constructor(props) {
+    super(props);
+
+    this.precisionRound = this.precisionRound.bind(this);
+  }
+
+  precisionRound(number, precision) {
+    var factor = Math.pow(10, precision)
+    return Math.round(number * factor) / factor
+  }
+
+  render() {
+    // No accounts found.
+    if (Object.keys(this.props.drizzleState.accounts).length === 0) {
+      return (
+        <span>Initializing...</span>
+      )
+    }
+
+    // Get account address and balance.
+    const address = this.props.drizzleState.accounts[this.props.accountIndex]
+    var balance = this.props.drizzleState.accountBalances[address]
```

Comment:
`var`?

Reply:
@adrianmcli All I did was copy over the old components and migrate them to the new context. I made a decision to leave the refactoring to later. Preferably after eslint and prettier have been put in. Thoughts?

Reply:
Yeah that makes sense.